circle.yml

```diff
@@ -1,16 +1,18 @@
 machine:
-  timezone: America/Los_Angeles
   php:
-    version: 5.5.0
+    version:
+      5.5.31
 
 dependencies:
   override:
     - mkdir -p build/logs
+    - echo "date.timezone = America/Los_Angeles" > /opt/circleci/php/$(phpenv global)/etc/conf.d/date.ini
     - composer update --prefer-stable --prefer-lowest --no-interaction
-    - sed -i 's/^;//' ~/.phpenv/versions/$(phpenv global)/etc/conf.d/xdebug.ini
+    - composer install-devtools
+    - phpenv versions
 
 test:
   override:
-    - phpunit
+    - devtools/vendor/bin/phpunit
```

Comment:
you have:
- adjusted PHP version on Circle CI
- added phpunit to dev tools
- install all dev tools

what about reverting all of those and update used PHP on CircleCI instead ? (to highest possible)